benchmark/Microsoft.IdentityModel.Benchmarks/ValidateTokenAsyncTests.cs

```diff
@@ -98,7 +99,16 @@ public async Task<TokenValidationResult> JsonWebTokenHandler_ValidateTokenAsyncW
             return await _jsonWebTokenHandler.ValidateTokenAsync(_jwsExtendedClaims, tokenValidationParameters).ConfigureAwait(false);
         }
 
-        [BenchmarkCategory("ValidateTokenAsyncWithTokenValidationParameters"), Benchmark]
+        [BenchmarkCategory("ValidateTokenAsync_Success"), Benchmark]
+        public async Task<bool> JsonWebTokenHandler_ValidateTokenAsyncWithVP()
+        {
+            // Because ValidationResult is an internal type, we cannot return it in the benchmark.
```

Comment:
do you want to open an issue to track this?

Reply:
Added #2797